### DISCUSSION TITLE
Mongock compatibility

### DISCUSSION THREAD

**Discussion by ghost** [flamingock/flamingock-java#324] [General]
Hello everyone,

Is Mongock v4.3.x compatible with Spring v2.4.x?

Cheers

**Comment by dieppa** [comment]
Hello @alhamidawi ,

here you can see the [compatibility table](https://www.mongock.io/compatibility-versions-table).

Although from version 4.3.8, Mongock uses Spring Boot version 2.3.7.RELEASE, I don't think you will have any problem using 2.4.X. 

However, with the new version 5, we'll be releasing around June, we'll provide our compatibility strategy with Spring to provide more clarity in an area that's always a bit tricky and less potential issues.

Cheers
